dropwizard-demo/test/com/test/ControllerTest.java

```diff
@@ -57,26 +54,8 @@ public void testcase_noTweetsFound() {
         Response actualTweet = tweetPost.GetTweets();
         Assert.assertEquals(expectedTweet.getEntity(), actualTweet.getEntity());
         Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
-        when(tweetPost.GetTweets()).thenReturn(Response.ok(str).build());
-        ArrayList<String> arrayList = new ArrayList<String>();
-        arrayList.add("hlo");
-        Response expectedTweet = Response.ok(arrayList).build();
-        Response actualTweet = tweetPost.GetTweets();
-        Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
-        Assert.assertEquals(expectedTweet.getStatus(),actualTweet.getStatus());
     }
 
-    @Test
-    public void testcase_noTweetsFound() throws TwitterException {
-        when(brsConfiguration.configurationBuilder()).thenReturn(new ConfigurationBuilder());
```

Comment:
Y these tests were deleted?

Reply:
This tests are added mutliple times, so deleted Pooja.

Reply:
cool